tests/phpunit/tests/file.php

```diff
@@ -444,7 +444,7 @@ public function test_file_signature_expected_failure() {
 		unlink( $file );
 
 		if ( is_wp_error( $verify ) && 'signature_verification_unsupported' === $verify->get_error_code() ) {
-			$this->markTestSkipped( 'This system does not support Signature Verification.' );
+			$this->fail( 'This system does not support Signature Verification.' );
```

Comment:
Same thing again. Failing tests should only be done when the code under test doesn't run correctly. Not when the environment in which the tests are run does not comply with the desired environment.